README.md

```diff
@@ -1,2 +1,165 @@
-# flask-rest-api
-RESTful API service using Python Flask
+# HashAPI
+RESTful API service using Python Flask.
+
+## API Service Utilized by hash_cli
+
+#### Description:
+The hash_api Python(3.6) flask web service API connects the hash_cli Client to a MongoDB database.  MongoDB is currently the only supported database for this web application.  MongoDB also must have user authentication enabled.  
+
+## Getting Started
+
+#### Install MongoDB via homebrew
+Install homebrew
+```
+install homebrew @ http://brew.sh/
```

Comment:
you could replace this with: _/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"_